### PR TITLE
Add TeamPostgreSQL (new formula)

### PR DESCRIPTION
**Pull request by jawshooah**

### CODE REVIEW

File: Library/Formula/teampostgresql.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+class Teampostgresql < Formula
+  homepage "http://www.teampostgresql.com"
+  url "http://cdn.webworks.dk/download/teampostgresql_multiplatform.zip"
```
Comment (jacknagel) [inline Library/Formula/teampostgresql.rb]: We need a non-moving URL (i.e. one with a version in it somewhere)

File: Library/Formula/teampostgresql.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    </plist>
+    EOS
+  end
+
+  test do
+    cmd = [
+      "/usr/bin/java",
+      "-cp", "#{lib}/webapp/WEB-INF/lib/*:#{lib}/webapp/WEB-INF/classes",
+      "dbexplorer.TeamPostgreSQL", "12345",
+      "#{(lib/"webapp").relative_path_from(Pathname.new(`/bin/pwd`))}"
```
Comment (jacknagel) [inline Library/Formula/teampostgresql.rb]: `Pathname.new(`/bin/pwd`)` can be `testpath`

File: Library/Formula/teampostgresql.rb [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    ].join(" ")
+
+    pipe_cmd_in, pipe_cmd_out = IO.pipe
+    cmd_pid = Process.spawn(cmd, [:err, :out] => pipe_cmd_out, :pgroup => true)
```
Comment (jacknagel) [inline Library/Formula/teampostgresql.rb]: Formula need to be compatible with Ruby 1.8, so `Process.spawn` is out, sorry.

File: Library/Formula/teampostgresql.rb (hunk 1 above)
Comment (jawshooah) [inline Library/Formula/teampostgresql.rb]: I've contacted the dev about it, waiting to hear back.

File: Library/Formula/teampostgresql.rb (hunk 3 above)
Comment (jawshooah) [inline Library/Formula/teampostgresql.rb]: Damn. What's a good legacy substitute? I'm fairly new to Ruby.

File: Library/Formula/teampostgresql.rb [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  sha1 "40ab16402ca8d862ae102a0ad44f84775dbe2a68"
+
+  depends_on :java => "1.7"
+
+  def install
+    rm_f Dir["*.bat", "*.sh"]
+    prefix.install_metafiles
+    lib.install Dir["*"]
+
+    libs    = lib/"webapp"/"WEB-INF"/"lib"
```
Comment (jawshooah) [inline Library/Formula/teampostgresql.rb]: @jacknagel Which syntax is preferred? This, or` "#{lib}/webapp/WEB-INF/lib"`?

File: Library/Formula/teampostgresql.rb [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  depends_on :java => "1.7"
+
+  def install
+    rm_f Dir["*.bat", "*.sh"]
+    prefix.install_metafiles
+    lib.install Dir["*"]
+
+    libs    = lib/"webapp"/"WEB-INF"/"lib"/"*"
+    classes = lib/"webapp"/"WEB-INF"/"classes"
+    main    = "dbexplorer.TeamPostgreSQL"
```
Comment (mikemcquaid) [inline Library/Formula/teampostgresql.rb]: Don't indent these `=` here, thanks.

Comment (jawshooah) [inline Library/Formula/teampostgresql.rb]: No problem, done in last commit.

**Comment by bfontaine** [comment]
Any update here?

**Comment by jawshooah** [comment]
Never got a response from the dev, and I'm no longer using this software. If someone else wants to pick up from here, they're welcome to.

**Pull request closed without merging**